utilities/compareScreenshots.js

```diff
@@ -0,0 +1,36 @@
+
+const fs = require('fs');
+const path = require('path');
+const imageDiff = require('image-diff');
+
+function compareScreenshots() {
+  const BASELINE_SCREENSHOTS_DIR = path.resolve('test', 'screenshots', 'baseline');
```

Comment:
with `path.resolve()` you can put unix path separators and they will be converted to the correct separators per platform. Then they read a bit nicer